packages/table/components/Table.tsx

```diff
@@ -122,6 +138,47 @@ export class Table<T> extends React.PureComponent<TableProps, TableState> {
     );
   }
 
+  private fillTableWidth(
+    colWidths: number[],
+    remainingWidth: number,
+    width: number
+  ) {
+    let filledWidth;
+    const totalWidth = colWidths.reduce((acc, curr) => acc + curr, 0);
+    const widthToFill = width - totalWidth;
+    const columns = React.Children.toArray(this.props.children) as Array<
+      React.ReactElement<ColumnProps>
+    >;
+    const fillWidthColsIdxArr: number[] = columns
+      .filter(child => child.props.growToFill)
+      .map(col => columns.indexOf(col));
+    const fillColumns = (colsIdxArr, widthToFill) => {
```

Comment:
let's please extract the inline function and unit-test separately

Reply:
so that imaginary utils module would consist of two functions `fillTableWidth` and `fillColumns`